tests/storage-compat/populate_db/populate_db.py

```diff
@@ -67,469 +97,760 @@ def upload_points(name: str):
+                    },
                 },
                 {
                     "id": 92,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"city": ["Berlin", "London"]},
                 },
                 {
                     "id": 93,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"city": ["Berlin", "Moscow"]},
                 },
                 {
                     "id": 94,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"city": ["London", "Moscow"]},
                 },
                 {
                     "id": 95,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"count": [0]},
                 },
                 {
                     "id": 96,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                 },
                 {
                     "id": 97,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"city": ["Berlin", "London"]},
                 },
                 {
                     "id": 98,
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"city": ["Berlin", "Moscow"]},
                 },
                 {
                     "id": "98a9a4b1-4ef2-46fb-8315-a97d874fe1d7",
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
                     "payload": {"city": ["London", "Moscow"]},
                 },
                 {
                     "id": "f0e09527-b096-42a8-94e9-ea94d342b925",
-                    "vector": rand_vec(),
+                    "vector": {
+                        "image": rand_dense_vec(),
+                        "text": rand_sparse_vec(),
+                    },
```

Comment:
This is A LOT of repetition, maybe a helper function might reduce it?

Reply:
Possible.

Also maybe we should be going for having more diversity
- some with only dense
- some with only sparse
- some with mix

:thinking:

Reply:
made everything better in https://github.com/qdrant/qdrant/pull/3210/commits/235a19cc1f4c2b835e1da6bc2dc24b5e9f53a315